headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -138,8 +136,10 @@ struct statuses {
     // 0x2E: Turns left until residual healing for the status in statuses::reflect, if applicable
     uint8_t reflect_damage_countdown;
     uint8_t curse; // 0x2F: STATUS_CURSED if 1
-    undefined field_0x30;
-    undefined field_0x31;
+    // 0x30: Set to monster::0x6 of the monster the attacker (the one causing the decoy status).
```

Comment:
Use the actual field names rather than offsets when possible, `monster::is_not_team_member`